doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-permissions-ref.adoc

```diff
@@ -0,0 +1,370 @@
+[id='security-management-rest-api-permissions-ref_{context}']
+= Permissions
+
+The Security Management REST API supports the following endpoints for managing permissions granted to the groups, roles and users in {CENTRAL}. The Security Management REST API base URL is `http://SERVER:PORT/business-central/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
+
+[GET] /groups/{groupName}/permissions::
++
+--
+Returns all permissions granted to a specified group.
+
+.Request parameters
+[cols="15%,45%,15%,25%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`groupName`
+|Name of the group whose permissions to be returned
```

Comment:
In this PR, search all instances of "whose". In each case, the grammar/structure seems off to my ear: ...whose __ to be __. I would suggest rewording to something like this:

Name of the group for whom you are retrieving permissions
Name of the user for whom you are retrieving assigned groups
Name of the user for whom you are retrieving assigned roles
Name of the user for whom you are changing the password
etc.

Not the greatest either, but it's consistent with the wording used in earlier sections (like "Name of the space for which you are retrieving projects" etc).

However, note that this is fine, for example:

Name of the user to be deleted